specs/posix/interface_ipv6.table

```diff
@@ -0,0 +1,13 @@
+table_name("interface_ipv6")
+description("IPv6 configuration and stats of network interfaces.")
+schema([
+    Column("interface", TEXT, "Interface name"),
+    Column("hlim", INTEGER, "Current Hop Limit"),
+    Column("forwarding", INTEGER, "Enable IP forwarding"),
+    Column("redirect", INTEGER, "Accept ICMP redirect messages"),
+    Column("rtadv", INTEGER, "Accept Router Advertisement"),
```

Comment:
May I suggest you to change the names of the columns to something more readable? E.g.: `interface` -> `interface_name`, `hlim` -> `hop_limit`, `forwarding` -> `forwarding_enabled`  and so on.

Reply:
`hlim` -> `hop_limit`
`forwarding` -> `forwarding_enabled`
`redirect` -> `redirect_accepted`
`rtadv` -> `rtadv_accepted`
Does this look reasonable?

I would like to keep the name `interface` (and not `interface_name`): the `routes`, `wifi`, `arp_cache`, and `interfaces_{addresses,details}` tables are using the name `interface`, so using `interface_name` for this table would break consistency and makes `JOIN` confusing.